awx/ui_next/src/components/Lookup/CredentialsLookup.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { withRouter } from 'react-router-dom';
+import { t } from '@lingui/macro';
+import { FormGroup, Tooltip } from '@patternfly/react-core';
+import { QuestionCircleIcon as PFQuestionCircleIcon } from '@patternfly/react-icons';
+import styled from 'styled-components';
+
+import { CredentialsAPI, CredentialTypesAPI } from '@api';
+import Lookup from '@components/Lookup';
+
+const QuestionCircleIcon = styled(PFQuestionCircleIcon)`
+  margin-left: 10px;
+`;
+
+class CredentialsLookup extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      credentials: props.credentials,
+      selectedCredentialType: { label: 'Machine', id: 1, kind: 'ssh' },
+      credentialTypes: [],
+    };
+
+    this.handleCredentialTypeSelect = this.handleCredentialTypeSelect.bind(
+      this
+    );
+    this.loadCredentials = this.loadCredentials.bind(this);
+    this.loadCredentialTypes = this.loadCredentialTypes.bind(this);
+    this.toggleCredential = this.toggleCredential.bind(this);
+  }
+
+  componentDidMount() {
+    this.loadCredentials({ page: 1, page_size: 5, order_by: 'name' });
```

Comment:
Why is this called here? This function doesn't do anything with the result plus `<Lookup>` below should be calling it directly (line 145)

Reply:
Good question.  Consider that function removed.